scripts/downloads/download_data.py

```diff
@@ -62,22 +62,25 @@ def download(self, save_dir: Path):
 
 
 @dataclass
-class FriendsDownload(DatasetDownload):
-    """Download the friends dataset."""
+class Sitcoms3DDownload(DatasetDownload):
+    """Download the sitcoms3D dataset."""
 
     def download(self, save_dir: Path):
-        """Download the friends dataset."""
+        """Download the sitcoms3D dataset."""
 
         # https://drive.google.com/file/d/1sgKr0ZO7BQC0FYinAnRSxobIWNucAST5/view?usp=sharing
-        friends_file_id = "1sgKr0ZO7BQC0FYinAnRSxobIWNucAST5"
+        sitcoms3d_file_id = "1sgKr0ZO7BQC0FYinAnRSxobIWNucAST5"
 
         # Download the files
-        url = f"https://drive.google.com/uc?id={friends_file_id}"
-        download_path = str(save_dir / "friends.zip")
+        url = f"https://drive.google.com/uc?id={sitcoms3d_file_id}"
+        download_path = str(save_dir / "sitcoms3d.zip")
         gdown.download(url, output=download_path)
         with zipfile.ZipFile(download_path, "r") as zip_ref:
             zip_ref.extractall(str(save_dir))
         os.remove(download_path)
+        # The folder name of the downloaded dataset is the previously using 'friends/'
+        if os.path.exists(str(save_dir / "friends/")):
+            os.rename(str(save_dir / "friends/"), str(save_dir / "sitcoms3d/"))
```

Comment:
Ah this is a bit annoying, but thanks for adding the comment for clarity and making it work. :)